appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -17,16 +18,18 @@ else{
 }
 
 function Rebase-Uri($originalURI){
-  $replace = $originalURI.IndexOf("file:///")
-  if($replace -ne  -1){
-      $caseSensitiveUri = Get-CaseSensitivePath($originalURI.SubString("file:///".Length))
-      $replaceLength = $caseSensitiveUri.IndexOf($projectSlug)
-      if($replaceLength -ne -1){
-          $builder.Path = ($repoName, $repoCommit, $caseSensitiveUri.SubString($replaceLength+$projectSlug.Length+1) -join '/')
-          return $builder.ToString()
-      }
+  if($originalURI){
+    $replace = $originalURI.IndexOf("file:///")
```

Comment:
But also, you only want to do this replacement if that string occurs at the _start_ of the URI. As written, the condition `if ($replace -ne -1)` will be true if that string occurs _anywhere_ in the URI. So change this to:

```
if ($originalUri.StartsWith($fileUriPrefix) {
```